lib/player.rb

```diff
@@ -0,0 +1,11 @@
+class Player
```

Comment:
Now I see the Player class is so bare. After it has been initialized it's not doing anything anymore. See some of my comments for the Game class